tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/JavaScriptLanguageSpecificChecks.cs

```diff
@@ -38,6 +38,33 @@ public async Task<CLICheckResponse> AnalyzeDependenciesAsync(string packagePath,
 
     public async Task<CLICheckResponse> UpdateSnippetsAsync(string packagePath, CancellationToken cancellationToken = default)
     {
-        return await Task.FromResult(new CLICheckResponse());
+        try
+        {
+            _logger.LogInformation("Running 'pnpm run update-snippets' in {PackagePath}", packagePath);
+
+            var result = await _processHelper.Run(new(
+                    "pnpm",
+                    ["run", "update-snippets"],
+                    workingDirectory: packagePath
+                ),
+                cancellationToken
+            );
+
+            if (result.ExitCode != 0)
+            {
+                _logger.LogError("'pnpm run update-snippets' failed with exit code {ExitCode}", result.ExitCode);
+                return new CLICheckResponse(result)
+                {
+                    NextSteps = ["Review the error output and attempt to resolve the issue."]
```

Comment:
@timovv with the current implementation this will have exit code 0, is that intended?